src/libs/getLocalFileUri/index.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Returns the URI string. For non-ios devices, we don't need to change anything
+ *  @param {bool} localFileUri which is fileUri coming in by default - from react native picker
+ * @return {String}
+ */
+
+export default localFileUri => localFileUri;
```

Comment:
Why can't we just use `decodeURIComponent` everywhere?

Reply:
@parasharrajat I believe I changed it because I was decoding the URL in another location. You are correct though, we can directly `decodeURIComponent` here. I made the change accordingly!